fcrepo-configs/src/main/java/org/fcrepo/config/OcflPropsConfig.java

```diff
@@ -133,6 +151,16 @@ private void postConstruct() throws IOException {
             LOGGER.info("Fedora OCFL S3 bucket: {}", ocflS3Bucket);
             LOGGER.info("Fedora OCFL S3 prefix: {}", ocflS3Prefix);
         }
+        FCREPO_DIGEST_ALGORITHM = DigestAlgorithm.fromAlgorithm(FCREPO_DIGEST_ALGORITHM_VALUE);
```

Comment:
I feel a little uneasy about moving `DigestAlgorithm` into `fcrepo-config`. The module is not really supposed to have "code" in it. What about moving it to `fcrepo-common` instead, and doing the digest resolution/validation logic inside `OcflPersistenceConfig`?

Reply:
`DigestAlgorithm` is an `enum`, much like [`Storage`](https://github.com/fcrepo/fcrepo/blob/main/fcrepo-configs/src/main/java/org/fcrepo/config/Storage.java) which already exists in `fcrepo-config` there which is why I thought that was allowed. 
I need to validate the configuration which is why I moved this into fcrepo-config, the DigestAlgorithm processing allows people more flexibility in the text they use (ie. sha512, SHA-512, sha-512). 

So I can do string comparisons or I can have a dependency on fcrepo-common (which I thought was a bad idea) if you'd like the `enum` moved

Reply:
That's fair. Let's leave it.